interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
```

Comment:
Yes, but not a great reason: this is old code.

I do need to fix *all* of those snake_case labels, but plan to do so when I'm refactoring because this will have a ripple effect into all of the other areas that touch them. I've made a reminder in #63, where I'll be doing that bit of refactoring.